### DISCUSSION TITLE
Daily.dev Card not working

### DISCUSSION THREAD

**Discussion by mrevjd** [dailydotdev/daily#385] [General]
When trying to generate a daily.dev card I see the following error in my browser:

Something went wrong, please try again...

Cheers

**Comment by idoshamun** [comment]
Hi @mrevjd,

Thanks for reporting. We're now investigating the issue

**Reply by idoshamun** [reply]
@mrevjd should be ok by now :)

**Reply by mrevjd** [reply]
Thanks for sorting this out. 
Working perfectly again.
